src/main/java/org/symphonyoss/integration/jira/authorization/JiraAuthorizationManager.java

```diff
@@ -327,11 +327,15 @@ public boolean isUserAuthorized(IntegrationSettings settings, String url, Long u
           provider.makeAuthorizedRequest(jiraOAuth1Data.getAccessToken(), myselfUrl,
               HttpMethods.GET, null);
 
-      return response.getStatusCode() != HttpStatusCodes.STATUS_CODE_UNAUTHORIZED;
+      return response.getStatusCode() == HttpStatusCodes.STATUS_CODE_OK;
     } catch (MalformedURLException e) {
       throw new JiraOAuth1Exception(MSG.getMessage("integration.jira.url.api.invalid", url),
           e, MSG.getMessage("integration.jira.url.api.invalid.solution"));
     } catch (OAuth1HttpRequestException e) {
+      if (HttpStatusCodes.STATUS_CODE_UNAUTHORIZED == e.getCode()) {
```

Comment:
Please test this error flow on the unit tests.

Reply:
Done.